src/create_disk.sh

```diff
@@ -486,7 +486,9 @@ ppc64le)
     ;;
 s390x)
     bootloader_backend=zipl
-    rdcore_zipl_args=("--boot-mount=$rootfs/boot" "--append-karg=ignition.firstboot")
+    # XXX: switch to using --append-karg once we have new enough rdcore
+    # https://github.com/coreos/coreos-installer/pull/950
```

Comment:
anything we can do to remind ourselves?

Reply:
Filed https://github.com/coreos/coreos-assembler/pull/3052.